### PR TITLE
Add Validation for Message

### PR DESCRIPTION
**Pull request by shrihari689**
## 💪 Changes

- Minor Change in Validation

🔨 Issue associated with the Frontend: https://github.com/coronasafe/care_fe/pull/2123

### CODE REVIEW

**Review by gigincg — CHANGES_REQUESTED** [review]
Can you also fix the spelling for message? In the error response

**Comment by shrihari689** [comment]
Updated @gigincg  🚀

**Comment by gigincg** [comment]
@shrihari689 Why did you remove  `or request.data["message"] == ""`

**Comment by shrihari689** [comment]
😲 I haven't removed it @gigincg 
📌 It's there on https://github.com/coronasafe/care/blob/ae1396463da27db295fb426658869700acc63416/care/facility/api/viewsets/notification.py#L57

File: care/facility/api/viewsets/notification.py [hunk 1]
```diff
@@ -52,10 +52,10 @@ class DummyNotificationSerializer(Serializer):  # Dummy for Spec
     @action(detail=False, methods=["POST"])
     def notify(self, request, *args, **kwargs):
         user = request.user
-        if "facility" not in request.data:
+        if "facility" not in request.data or request.data["facility"] == "":
```
Comment (vigneshhari) [inline care/facility/api/viewsets/notification.py:55]: Need more solid validation to check if the value is a UUID, but we'll build it later on.

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**